docs/reference-guides/data/data-core-editor.md

```diff
@@ -1241,6 +1253,19 @@ _Related_
 
 -   selectBlock in core/block-editor store.
 
+### setRenderingMode
```

Comment:
`renderingMode` is as good a name as any. I appreciate how you've combined `pageContentFocusType` values 👍🏻

Reply:
Same, it's a nice way of combining things. I wonder if it'd be worth making it a tiny bit more specific and using something like `blockRenderingMode`? Mostly I'm trying to think how can the name hint that this mode is about which blocks are displayed (and their locked status) 🤔

Reply:
I prefer slightly renderingMode over blockRenderingMode because blockRenderingMode feels like "block specific" while here we're just actually focusing on one entity over another for both content (blocks) but also things like sidebars, headers...

Reply:
Good point! Sounds reasonable to me 🙂